### PR TITLE
trans-Why-Kubernetes-Doesnt-Use-Libnetwork

### PR DESCRIPTION
**Pull request by hwdef**

### CODE REVIEW

**Comment by hwdef** [comment]
@xichengliudui

File: content/zh/blog/_posts/2016-01-00-Why-Kubernetes-Doesnt-Use-Libnetwork.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+date: 2016-01-14
+slug: why-kubernetes-doesnt-use-libnetwork
+url: /blog/2016/01/Why-Kubernetes-Doesnt-Use-Libnetwork
+--- -->
+
+<!-- Kubernetes has had a very basic form of network plugins since before version 1.0 was released — around the same time as Docker's [libnetwork](https://github.com/docker/libnetwork) and Container Network Model ([CNM](https://github.com/docker/libnetwork/blob/master/docs/design.md)) was introduced. Unlike libnetwork, the Kubernetes plugin system still retains its "alpha" designation. Now that Docker's network plugin support is released and supported, an obvious question we get is why Kubernetes has not adopted it yet. After all, vendors will almost certainly be writing plugins for Docker — we would all be better off using the same drivers, right?   -->
+
+在 1.0 版本发布之前，Kubernetes 已经有了一个非常基础的网络插件形式-大约在引入 Docker’s [libnetwork](https://github.com/docker/libnetwork) 和 Container Network Model ([CNM](https://github.com/docker/libnetwork/blob/master/docs/design.md)) 的时候。与 libnetwork 不同，Kubernetes 插件系统仍然保留它的 'alpha' 名称。现在 Docker 的网络插件支持已经发布并得到支持，我们发现一个明显的问题是 Kubernetes 尚未采用它。毕竟，供应商几乎肯定会为 Docker 编写插件-我们最好还是用相同的驱动程序，对吧？
```
[the remark this answers is not in the document]
Comment (hwdef) [reply content/zh/blog/_posts/2016-01-00-Why-Kubernetes-Doesnt-Use-Libnetwork.md]: docker's 应该是跟后面的libnetwork连起来的吧。

[the remark this answers is not in the document]
Comment (xichengliudui) [reply content/zh/blog/_posts/2016-01-00-Why-Kubernetes-Doesnt-Use-Libnetwork.md]: There is no such form in Chinese

[the remark this answers is not in the document]
Comment (hwdef) [reply content/zh/blog/_posts/2016-01-00-Why-Kubernetes-Doesnt-Use-Libnetwork.md]: emm sorry about that,
Docker's libnetwork,
if Docker's ->Docker, there is a typo.

**Pull request merged**